crypto/jwk_test.go

```diff
@@ -14,12 +14,12 @@ func TestJWKToPrivateKeyJWK(t *testing.T) {
 	assert.NotEmpty(t, privateKey)
 
 	// convert to JWK
-	jwk, err := jwk.New(privateKey)
+	jwkKey, err := jwk.New(privateKey)
```

Comment:
jwk means json web key, so this may be repetitive

Reply:
key alone is fine

Reply:
Done